Packs/Carbon_Black_Enterprise_Live_Response/Playbooks/playbook-Get_File_Sample_From_Path_-_VMware_Carbon_Black_EDR_-_Live_Response_API_README.md

```diff
@@ -0,0 +1,50 @@
+This playbook retrieve file from endpoints by path using VMware Carbon Black EDR (Live Response API).
+Make sure to provide Carbon Black sensor ID of the endpoint from which you want to retrieve the file.
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+* GenericPolling
+
+### Integrations
+* carbonblackliveresponse
+* integration-Carbon_Black_Enterprise_Live_Response
+
+### Scripts
+* Exists
+
+### Commands
+* cb-get-file-from-endpoint
+* cb-session-create
+* cb-list-sessions
+* cb-session-close
+
+## Playbook Inputs
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| Sensor_ID | The sensor ID. Provided the sensor ID to run the command with a new session. The session will be created and closed automatically. |  | Optional |
+| Path | The path of the file to retrieve.<br/>For example:<br/>C:\\users\\folder\\file.txt |  | Optional |
```

Comment:
As we said double quotes are not mandatory
Better to change not to confuse customers.